h2o-py/tests/testdir_algos/automl/pyunit_automl_args.py

```diff
@@ -181,6 +181,16 @@ def prostate_automl_args():
     assert amodel.params['keep_cross_validation_fold_assignment']['actual'] == False
     assert amodel._model_json["output"]["cross_validation_fold_assignment_frame_id"] == None
 
+
+    print("Check that automl gets interrupted after `max_runtime_secs`")
+    max_runtime_secs = 30
```

Comment:
I don't think we had issues before with 30 sec runtime. Can try to increase this (if okay for the test harness) to really see if there are any changes.

Reply:
I'm just concerned if we want to add more tests today that wait for >5mn to complete, as part of the standard build pipeline; those would make more sense only for nightly build tests (if those did exist).
Erin and me ran individual tests with various timeouts against this branch, but if we want to add them as part of the standard build, I'm fine.

Reply:
I think having nightly runs of AutoML are necessary at this point. Especially if something like this can potentially occur in the future. Thoughts @ledell ?